tests/lib/nested.sh

```diff
@@ -164,6 +164,7 @@ get_nested_snap_channel(){
 
 get_image_url_for_nested_vm(){
     if [[ "$SPREAD_BACKEND" == google* ]]; then
+        #shellcheck disable=SC2119
```

Comment:
Why is this override necessary, we are not passing any arguments here.

Reply:
Yeah, let's remove this one if we don't need it.